src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -203,10 +258,12 @@ class FunctionBuilder[F >: Null](parameterTypeInfo: Array[MaybeGenericTypeInfo[_
 
   def result(print: Option[PrintWriter] = None): () => F = {
     val bytes = classAsBytes(print)
-    val localName = name.replaceAll("/",".")
+    val localName = name.replaceAll("/", ".")
 
     new (() => F) with java.io.Serializable {
-      @transient @volatile private var f: F = null
+      @transient
+      @volatile private var f: F = null
```

Comment:
boo

Reply:
Sorry, I'll fix. Unless you weren't talking about the whitespace...